src/proof/dot/dot_printer.cpp

```diff
@@ -25,55 +25,103 @@
 namespace cvc5 {
 namespace proof {
 
-void DotPrinter::cleanQuotes(std::string& s)
+// guarantee that string does not have unescaped quotes
+std::string DotPrinter::sanitizeString(const std::string& s)
 {
-  std::string rep("\\\"");
-  for (size_t pos = 0; (pos = s.find("\"", pos)) != std::string::npos;
-       pos += rep.length())
+  std::string newS;
+  newS.reserve(s.size());
```

Comment:
why do you reverse?

Reply:
At that moment I know the minimum size of the string, so I allocate memory to it.

Reply:
I misread reserve or reverse, sorry :)